lang/javascript.py

```diff
@@ -1,120 +1,101 @@
 from talon.voice import Context, Key, press
 import talon.clip as clip
-from ..utils import (
-    text,
-    parse_words,
-    parse_words_as_integer,
-    insert,
-    word,
-    join_words,
-    is_filetype,
-)
+from ..utils import text, parse_words, parse_words_as_integer, insert, word, join_words
 
-JS_EXTENSIONS = (".js", ".jsx")
-
-context = Context("javascript", func=is_filetype(JS_EXTENSIONS))
+def verifyExtension(app, win):
+    return win.doc.endswith(".js") or win.doc.endswith(".jsx")
 
+context = Context("javascript")
 
 def remove_spaces_around_dashes(m):
     words = parse_words(m)
-    s = " ".join(words)
-    s = s.replace(" – ", "-")
+    s = ' '.join(words)
```

Comment:
it looks like this code still needs to be run through `black` formatter

Reply:
Not sure what you mean by this. Sorry.

Reply:
we are using the python automatic formatting tool called `black` https://black.readthedocs.io/en/stable/ You should be able to pip install it and run on all your code `black *` or so.

Reply:
Okay. My Python knowledge is next to zero, but I will try to figure that out.

Reply:
Dont worry about it. Let me know when it's ready and I can run it